### ISSUE TITLE
Error while parsing SQL

### ISSUE THREAD

**Issue by DrSteam1111** [azerothcore/mod-individual-xp#30]
Running Induvidual-xp on mysql server version `8.0.27.0`
gives you the following error in the worldserver

```
[1064] You have an error in your SQL syntax; check the manual that corresponds to your 
MySQL server version for the right syntax to use near '%u' at line 1 Error while parsing SQL. 
Core fix required.
```

**Comment by jllllll** [comment]
This is due to the incorrect usage of %u at lines 70 and 87. It should be '{}' instead of '%u'.

**Comment by jllllll** [comment]
This is now fixed with a36b3a1.

**Comment by pangolp** [comment]
At present, this issue has been resolved. If I am wrong, please let me know. Thank you.
